ERROR: role-to-test/filter_plugins/network_engine.py:105:1: E303 too many blank
ERROR: role-to-test/filter_plugins/network_engine.py:84:14: undefined-variable Undefined variable 'FilterError'

--- a/filter_plugins/network_engine.py
+++ b/filter_plugins/network_engine.py
@@ -81,7 +81,7 @@ def vlan_expand(vlan):
 
     match = re.match(r'([A-Za-z]*)(.+)', vlan)
     if not match:
-        raise FilterError('unable to parse vlan %s' % vlan)
+        raise AnsibleFilterError('unable to parse vlan %s' % vlan)
 
     index = match.group(2)
     indices = list()
@@ -99,7 +99,6 @@ def vlan_expand(vlan):
                 i += 1
 
     return ['%d' % int(index) for index in indices]
-
 
 
 class FilterModule(object):

